Source/Core/Core/DSP/DSPCore.h

```diff
@@ -308,6 +308,9 @@ struct SDSP
 
   // This one doesn't really belong here.
   u8* cpu_ram;
+
+  // This is used by DSPJitRegCache to store EBP.
+  u64 ebp_store;
```

Comment:
Ah. Good point.

Thinking about it one could probably completely get rid of this special handling given that it is unnecessary in the x86_64 ABI, unlike x86 which this was probably originally intended for.